node_modules/oae-core/linkpreview/js/linkpreview.js

```diff
@@ -135,8 +135,10 @@ define(['jquery', 'oae.core', 'jquery.oembed'], function($, oae) {
                     embeddable = false;
                 }
             }
+            var supportedTypes = _.union(_.pluck(navigator.mimeTypes, 'type'), ['text/html', 'text/plain']);
```

Comment:
@stuartf : When the browser supports showing a PDF file on the screen, will this catch it and let it render on the screen? What if the headers of the request specify that the file has to be downloaded?